samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -297,3 +300,104 @@ SnapToCenterVirtualGridList.storyName = 'Snap to center VirtualGridList';
 SnapToCenterVirtualGridList.parameters = {
 	propTables: [Config]
 };
+
+class VirtualGridListInScroller extends Component {
+	renderItem = ({index, ...rest}) => {
+		const
+			color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
+			source = {
+				'hd': `http://placehold.it/200x200/${color}/ffffff&text=Image ${index}`,
+				'fhd': `http://placehold.it/300x300/${color}/ffffff&text=Image ${index}`,
+				'uhd': `http://placehold.it/600x600/${color}/ffffff&text=Image ${index}`
+			},
+			text = `Item ${index}`;
+
+		return (
+			<ImageItem {...rest} label={text} onClick={this.props.onNext} src={source} />
+		);
+	};
+
+	render () {
+		const props = Object.assign({}, this.props);
+		delete props.onNext;
+
+		const virtualGridListProps = {
+			dataSize: updateDataSize(number('dataSize', Config, defaultDataSize)),
+			direction: 'horizontal',
+			itemRenderer: this.renderItem,
+			itemSize: {
+				minWidth: ri.scale(number('minWidth', Config, 688)),
+				minHeight: ri.scale(number('minHeight', Config, 570))
+			},
+			spacing: ri.scale(number('spacing', Config, 0)),
+			style: {
+				height: ri.scale(number('minHeight', Config, 570)),
+				paddingBottom: ri.unit(ri.scale(36) + 'px', 'rem')
```

Comment:
We have `ri.scaleToRem` function for this.

Reply:
It's fixed.